### PR TITLE
feat: set real data to SeverityStatusByService widget

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SRD(Set Real Data😎)

스크린샷 2023-06-12 오후 1 00 25

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue [hunk 1]
```diff
@@ -38,48 +39,115 @@ import {
...
         end: dayjs.utc(state.settings?.date_range?.end).format(DATE_FORMAT),
     })),
-    sampleData: computed(() => [...Array(20)].map(() => Object.values(SEVERITY_STATUS_MAP)[Math.floor(Math.random() * 5)])),
     boxWidth: computed<number>(() => {
         if (!props.width) return 112;
         const widgetPadding = 24;
         const widgetContentWidth = props.width - (widgetPadding * 2);
         if (props.width >= 990) return widgetContentWidth / 8;
-        return widgetContentWidth / 7;
+        return widgetContentWidth / 7 < 112 ? 112 : widgetContentWidth / 7;
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:79]: question: What does number '112' mean? 😮

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue [hunk 2]
```diff
@@ -38,48 +39,115 @@ import {
...
+                page: {
+                    limit: 80,
+                    start: 1,
+                },
+                ...apiQueryHelper.data,
+            },
+        });
+        return refineData(results);
+    } catch (e) {
+        console.error(e);
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:122]: suggestion: Please use `ErrorHandler`.

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue [hunk 3]
```diff
@@ -38,48 +39,115 @@ import {
...
+            .filter((s) => s.priority === minSeverityPriority)[0].name; // ex. 'CRITICAL'
+        const complianceStatus = result.status.includes('FAIL') ? 'FAIL' : 'PASS';
+        return {
+            service: result.service,
+            severity,
+            complianceStatus,
+            value: result.fail_finding_count,
+        };
+    });
 };
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:143]: nitpick: It was pretty hard to read for me. 🥲
What about separate the process of getting the exact priority's severity name?

```suggestion
const SEVERITY_PRIORITY_MAP: Record<number, typeof SEVERITY_STATUS_MAP[Severity]> = {}
Object.values(SEVERITY_STATUS_MAP).forEach(s => { SEVERITY_STATUS_MAP[s.priority] = s })

const refineData = (data): Data[] => {
    const { results } = data;
    return results.map((result) => {
        const minSeverityPriority = min(result.severity.map((s) => SEVERITY_STATUS_MAP[s].priority)); // ex. 1
        const severity = SEVERITY_PRIORITY_MAP[minSeverityPriority]?.name // ex. 'CRITICAL'
        const complianceStatus = result.status.includes('FAIL') ? 'FAIL' : 'PASS';
        return {
            service: result.service,
            severity,
            complianceStatus,
            value: result.fail_finding_count,
        };
    });
};
```

**Review by WANZARGEN — APPROVED** [review]
Gooood!!!! I left small comments!!

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue (hunk 2 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:122]: oh!! thanks

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:79]: It's min width of box! I'll replace it to constant value :)

**Review by skdud4659 — APPROVED** [review]
LGTM!!!!!!!!!!

**Pull request merged**